trailblazer/constants.py

```diff
@@ -111,7 +110,16 @@ def list(cls):
     "failed": TrailblazerStatus.FAILED,
 }
 
-TOWER_TIMESTAMP_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
-TOWER_TIMESTAMP_FORMAT_ALTERNATIVE = "%Y-%m-%dT%H:%M:%S.%fZ"
-
 STATUS_OPTIONS = tuple(TrailblazerStatus.list())
+ONGOING_STATUSES = (
+    TrailblazerStatus.PENDING,
+    TrailblazerStatus.RUNNING,
+    TrailblazerStatus.COMPLETING,
+    TrailblazerStatus.ERROR,
+)
+JOB_STATUS_OPTIONS = tuple(SlurmJobStatus.list())
```

Comment:
Same here I get `<property object at 0x7f9ce823e5e0>` instead of the actual value when using properties. I could however replace the `list` classmethod by a `statuses` classmethod. Would that be better?

Reply:
It is better, since it is harder to understand what list returns, except a list but a list of what? statues tells the story

Reply:
replaced and removed status constants